### ISSUE TITLE
BehaviorSubject.scan not picking up initial value

### ISSUE THREAD

**Issue by mbrandonw** [ReactiveX/RxJava#3168]
I would expect this:

```
final Observable<Void> ob = BehaviorSubject.create();
ob.scan(0, (accum, x) -> 123)
  .subscribe(x -> {
    Timber.d("Hit!");
  });
```

to emit a value (and indeed it does in the equivalent RxJS code).

Is this a bug?

**Comment by akarnokd** [comment]
This is odd. The initial value is emitted just before the first accumulated value or a completion event.

**Comment by akarnokd** [comment]
Fix available via #3171.
